images/pull-test-infra-gubernator/runner

```diff
@@ -19,8 +19,7 @@ set -o pipefail
 
 git clone https://github.com/kubernetes/test-infra
 ./test-infra/jenkins/bootstrap.py \
-    --repo=k8s.io/${REPO_NAME} \
+    --repo=k8s.io/${REPO_NAME}=${PULL_REFS} \
```

Comment:
I would suggest removing all these flags from the runner and just relying on $@
all this "help" is a maintenance nightmare.

Reply:
I agree but I don't actually see where this image is being used yet. @rmmh ?

Reply:
er nevermind, grepped in the images dir :roll_eyes: I'll do this and bump.

Reply:
done. bump next